stm/stm32f4xx/stm32f4xx_hal_init.c

```diff
@@ -4,7 +4,7 @@
  * @brief This function is executed in case of error occurrence.
  * 
  */
-static void Error_Handler(void)
+void Error_Handler(void)
 {
   __disable_irq();
   while (1)
```

Comment:
Perhaps this should be kept static and file-local, and a completely different error_handler could be used for ADC (could call it adc_error_handler and keep it file-local to ADC as well). See my comments on the need for the ADC error-handler as well